[CS2103-T16-4] guiltTrip()

rongrongrr opened the pull request:
@jessicax941
@SwampertX
@yuhongtay
@Dandford
@rongrongrr

moziliar reviewed (commented):
Logic component diagram is not updated.
Stats should be shown in the model diagram.
Design alternatives for some of the implementations are missing. Might need to add them in.
Reminder class diagram has typo.
The rest are easy to understand.

jessicax941 committed:
Fix merge conflicts

jessicax941 committed:
Comment out failing tests (temporary)

jessicax941 committed:
Merge branch 'v1.4-UI' of https://github.com/jessicax941/main into v1.4-UI

jessicax941 committed:
Fix checkstyle issues

Dandford committed:
Merge pull request #261 from jessicax941/v1.4-UI

Finalise UI for v1.4, Add some tests for Income

jessicax941 committed:
Fix error on startup

rongrongrr committed:
Comment out EntryListPanel

yeejian-tan committed:
Hack hack hack

yeejian-tan committed:
More hacks

Dandford committed:
Update more tests

yuhongtay committed:
update

jessicax941 committed:
Merge pull request #273 from SwampertX/as-hack

Hack hack hack

Dandford committed:
Adding tests for CRUD for Category

Dandford committed:
Pulling from upstream

yuhongtay committed:
Modified edit reminder command for general reminder

jessicax941 committed:
Increase font size of application

jessicax941 committed:
Merge remote-tracking branch 'upstream/master' into v1.4-UI

jessicax941 committed:
Modify Statistics UI, add more sample data entries

rongrongrr committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103-T16-4/main

# Conflicts:
#	src/main/java/seedu/guilttrip/logic/commands/ListBudgetCommand.java
#	src/main/java/seedu/guilttrip/ui/MainWindow.java

rongrongrr committed:
Merge pull request #16 from AY1920S1-CS2103-T16-4/master

Merge Updates

rongrongrr committed:
Edit Budget progress bar to show spent amount instead of left amount. Edit progress bar to show red if spent exceeds budget amount else show blue

yeejian-tan committed:
Enable d/m kind of date entry

yeejian-tan committed:
Allow expressive error messages for add commands

jessicax941 committed:
Add TestFX libray, GUI handle test files

rongrongrr committed:
Edit Budget function to stop filtering Expense when implementing addBudget

jessicax941 committed:
Add Expense and Income GUI handles

jessicax941 committed:
Add Autoexpense, Wish and Budget GUI handles

jessicax941 committed:
Allow listAllReminders to switch out main panel with reminders

Modify WishList command to ListWish

jessicax941 committed:
Merge pull request #282 from rongrongrr/master

Edited Budget and Budget Display

jessicax941 committed:
Modify MainWindow.java

Change message for List command

jessicax941 committed:
Fix merge conflicts

yeejian-tan committed:
Allow expressive error messages for add commands

Also comment out tests that check for error messages
to be same for all errors.

jessicax941 committed:
Modify BudgetCard so spent amount turns red when budget is exceeded

yeejian-tan committed:
Merge branch 'master' into qol-fixes

yeejian-tan committed:
Easter egg!

When you type "uwu"

jessicax941 committed:
Merge pull request #284 from SwampertX/qol-fixes

Qol fixes

jessicax941 committed:
Merge branch 'master' into master

Dandford committed:
Adding tests for statistics

Dandford committed:
Merge branch 'master' of github.com:AY1920S1-CS2103-T16-4/main

jessicax941 committed:
WIP: Make Popup window for Reminders and add UI test util

yuhongtay committed:
Waiting for undo redo compatibility

Dandford committed:
Merge pull request #277 from Dandford/master

Adding even more tests for CRUD For Category

Dandford committed:
Adding Category Type Enum

Dandford committed:
Merge branch 'master' of github.com:Dandford/main

Dandford committed:
Fix Checkstyle Errors

yeejian-tan committed:
Forks expense creation from autoExp to background

but javafx throws a lot of errors.

rongrongrr committed:
Edit EditBudgetCommand to be able to edit budget's category

yeejian-tan committed:
Allow each AutoExp to track expenses created

jessicax941 committed:
Add UI tests for CommandBox, ResultDisplay, ExpenseCard, ExpenseListPanel

jessicax941 committed:
Set up reminder popup alert, Modify Theme

Dandford committed:
Modified Category Enum

jessicax941 committed:
Fix checkstyle issues, add more tests for UI

jessicax941 committed:
Fix checkstyle issues

jessicax941 committed:
Fix merge conflicts

jessicax941 committed:
Add BudgetBuilder, tests for BudgetPanel and BudgetCard

jessicax941 committed:
Add AutoExpenseCard, AutoExpenseListPanel tests

jessicax941 committed:
Fix checkstyle issues

jessicax941 committed:
Implement PanelName as Enum, Modify ToggleCommand usage message

jessicax941 committed:
Increase result display size

Dandford committed:
Reverting to before YearList

jessicax941 committed:
Convert FontName to Enum and fix some bugs

Fix changeFont bug, modify changeFont command message to show list of fonts and remove listFont command

jessicax941 committed:
Modify spacings and paddings and add border for Expenses and Incomes panel

Dandford committed:
Updating Dev-Guide Diagrams

Dandford committed:
Merge branch 'master' of github.com:AY1920S1-CS2103-T16-4/main

jessicax941 committed:
Fix bug on startup regarding JsonAdaptedBudget

jessicax941 committed:
Fix bug: reminders side panel title changing of colour

jessicax941 committed:
Fix checkstyle issues, temporarily comment out failing tests

yeejian-tan committed:
Fix AddExpense Error Message

jessicax941 committed:
Fix import checkstyle issues

yeejian-tan committed:
Merge branch 'master' into qol-fixes

jessicax941 committed:
Fix checkstyle issues

jessicax941 committed:
Merge pull request #289 from SwampertX/qol-fixes

Minor bug fixes

jessicax941 committed:
Merge remote-tracking branch 'upstream/master' into v1.4-UI

Dandford committed:
updating UserGuide

Dandford committed:
Merge branch 'master' of github.com:AY1920S1-CS2103-T16-4/main

jessicax941 committed:
Modify listBudget and listWish so that only budgets/wishes appear on main panel

jessicax941 committed:
Comment out JsonUserPrefsStorageTest and ConfigUtilTest to check if travis passes

jessicax941 committed:
Temporarily comment out AppParametersTest.java

jessicax941 committed:
Modify travis to run UI tests

yuhongtay committed:
Implemented reminders without visuals

jessicax941 committed:
Temporarily comment out failing tests

jessicax941 committed:
Fix checkstyle issues

jessicax941 committed:
Increase time allowance for performance test for UI

Dandford committed:
Completed Dev-Guide

Dandford committed:
Merge pull request #283 from jessicax941/v1.4-UI

Modify UI, add UI Tests

jessicax941 committed:
Modify switch theme method in MainWindow.java

jessicax941 committed:
Remove listFont related methods and attributes from CommandResult and MainWindow

Dandford committed:
Fixing broken Expense Tests

jessicax941 committed:
Update sequence diagram for TogglePanelCommand

jessicax941 committed:
Update implementation and activity diagram of toggle panel command

yuhongtay committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103-T16-4/main

# Conflicts:
#	src/main/java/seedu/guilttrip/logic/commands/CommandResult.java
#	src/main/java/seedu/guilttrip/logic/commands/conditioncommands/AddClassConditionCommand.java
#	src/main/java/seedu/guilttrip/logic/commands/conditioncommands/AddDateConditionCommand.java
#	src/main/java/seedu/guilttrip/logic/commands/conditioncommands/AddHasKeyWordConditionCommand.java
#	src/main/java/seedu/guilttrip/logic/commands/conditioncommands/AddQuotaConditionCommand.java
#	src/main/java/seedu/guilttrip/logic/commands/conditioncommands/AddTagsConditionCommand.java
#	src/main/java/seedu/guilttrip/logic/commands/conditioncommands/DeleteConditionCommand.java
#	src/main/java/seedu/guilttrip/logic/commands/conditioncommands/ReplaceConditionCommand.java
#	src/main/java/seedu/guilttrip/logic/commands/conditioncommands/ShowConditionListCommand.java
#	src/main/java/seedu/guilttrip/logic/commands/remindercommands/AddConditionToReminderCommand.java
#	src/main/java/seedu/guilttrip/logic/commands/remindercommands/AddReminderCommand.java
#	src/main/java/seedu/guilttrip/logic/commands/remindercommands/DeleteReminderCommand.java
#	src/main/java/seedu/guilttrip/logic/commands/remindercommands/EditReminderCommand.java
#	src/main/java/seedu/guilttrip/logic/commands/remindercommands/ListActiveRemindersCommand.java
#	src/main/java/seedu/guilttrip/logic/commands/remindercommands/RemoveConditionFromReminderCommand.java
#	src/main/java/seedu/guilttrip/logic/commands/remindercommands/SetEntryReminderCommand.java
#	src/main/java/seedu/guilttrip/logic/parser/GuiltTripParser.java
#	src/main/java/seedu/guilttrip/logic/parser/ParserUtil.java
#	src/main/java/seedu/guilttrip/logic/parser/remindercommandparsers/EditReminderCommandParser.java
#	src/main/java/seedu/guilttrip/logic/parser/remindercommandparsers/SetEntryReminderCommandParser.java
#	src/main/java/seedu/guilttrip/model/GuiltTrip.java
#	src/main/java/seedu/guilttrip/model/Model.java
#	src/main/java/seedu/guilttrip/model/ModelManager.java
#	src/main/java/seedu/guilttrip/model/entry/SortType.java
#	src/main/java/seedu/guilttrip/model/reminders/Reminder.java
#	src/main/java/seedu/guilttrip/model/reminders/conditions/Condition.java
#	src/main/java/seedu/guilttrip/ui/MainWindow.java
#	src/main/java/seedu/guilttrip/ui/NotificationPanel.java
#	src/main/java/seedu/guilttrip/ui/ReminderPanel.java
#	src/main/java/seedu/guilttrip/ui/reminder/ReminderCard.java
#	src/main/java/seedu/guilttrip/ui/reminder/ReminderPanel.java
#	src/test/java/seedu/guilttrip/logic/commands/CommandTestUtil.java
#	src/test/java/seedu/guilttrip/testutil/TypicalEntries.java

jessicax941 committed:
Modify font variable in MainWindow

yuhongtay committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103-T16-4/main

# Conflicts:
#	src/main/java/seedu/guilttrip/MainApp.java
#	src/main/java/seedu/guilttrip/logic/commands/CommandResult.java
#	src/main/java/seedu/guilttrip/logic/commands/uicommands/TogglePanelCommand.java
#	src/main/java/seedu/guilttrip/ui/MainWindow.java
#	src/main/java/seedu/guilttrip/ui/util/PanelName.java
#	src/test/java/seedu/guilttrip/logic/commands/FindExpenseCommandTest.java

Dandford committed:
updating User Guide

Dandford committed:
Fixing checkstyleErrors

jessicax941 committed:
Update UI class diagram

yuhongtay committed:
Rename DisplayPopUpCommand.java to DisplayPopupCommand.java

Rename file name.

yuhongtay committed:
fix eof

yuhongtay committed:
Merge branch 'master' of https://github.com/yuhongtay/main

yuhongtay committed:
checkstyle fixes

yuhongtay committed:
checkstyle fixes

jessicax941 committed:
Add ChangeFont sequence diagram and implementation for ChangeFont command

jessicax941 committed:
Modify TogglePanel activity diagram

yuhongtay committed:
checkstyle fix

jessicax941 committed:
Add activity diagram for ChangeFont command

jessicax941 committed:
Add activity diagram for ChangeFont command

yuhongtay committed:
test case fixes

yuhongtay committed:
checkstyle fixes

jessicax941 committed:
Add activity and sequence diagrams and implementation for SetLightTheme command in DG

jessicax941 committed:
Fix bug in Set Dark/Light Theme

Dandford committed:
Update User Guide

yuhongtay committed:
fix more bugs + test cases

jessicax941 committed:
Merge branch 'master' into update-DG

jessicax941 committed:
Update UI features section in UG

Dandford committed:
Fix Checkstyle

Dandford committed:
Modifying User Guide

yuhongtay committed:
Implement Ui

jessicax941 committed:
Add screenshots to UI features section in UG

Dandford committed:
Fixing errors

Dandford committed:
Modifying User Guide

jessicax941 committed:
Rename prefixes to original prefixes

Dandford committed:
Added PNG

Dandford committed:
Checking PPP

Dandford committed:
PPP

jessicax941 committed:
Add partial UI class diagram for toggle panel command

jessicax941 committed:
Merge remote-tracking branch 'upstream/master' into update-DG

jessicax941 committed:
Merge branch 'update-DG' of https://github.com/jessicax941/main into update-DG

jessicax941 committed:
Fix import checkstyle issue

Dandford committed:
Checking for PPP

jessicax941 committed:
Fix error in StorageManagerTest.java

Dandford committed:
Modifying Guide

jessicax941 committed:
Add PPP

jessicax941 committed:
Modify PPP and DG

Dandford committed:
Updating User Guide

yuhongtay committed:
Update DeveloperGuide.adoc

Dandford committed:
Updating User Guide Again

jessicax941 committed:
Update Ui.png

yuhongtay committed:
Update DeveloperGuide.adoc

yuhongtay committed:
Add files via upload

yuhongtay committed:
Update DeveloperGuide.adoc

yeejian-tan committed:
Add AutoSuggest UserGuide with screenshots

jessicax941 committed:
Add instructions for manual testing in DG

yeejian-tan committed:
Add screenshots for AutoExpense in UG

yuhongtay committed:
Update DeveloperGuide.adoc

yuhongtay committed:
Update DeveloperGuide.adoc

Dandford committed:
Modifying User Guide

yuhongtay committed:
final update

yuhongtay committed:
Update DeveloperGuide.adoc

remove whitespace

Dandford committed:
Update Dev Guide

yuhongtay committed:
Update DeveloperGuide.adoc

Dandford committed:
Updating User Guide. (#293)

* Adding tests for statistics

* Adding Category Type Enum

* Fix Checkstyle Errors

* Modified Category Enum

* Reverting to before YearList

* Updating Dev-Guide Diagrams

* updating UserGuide

* Completed Dev-Guide

* updating User Guide

* Fixing checkstyleErrors

* Update User Guide

* Fix Checkstyle

* Modifying User Guide

* Fixing errors

* Modifying User Guide

* Added PNG

* Checking PPP

* PPP

* Checking for PPP

* Modifying Guide

* Updating User Guide

* Updating User Guide Again

yuhongtay committed:
Update DeveloperGuide.adoc

Remove whitespace

yuhongtay committed:
Update DeveloperGuide.adoc

Dandford committed:
Fix User Guide Issues

Dandford committed:
Updating User Guide

jessicax941 committed:
Fix merge conflicts

Dandford committed:
Update PPP

Dandford committed:
Updating PPP Again

yuhongtay committed:
Update UserGuide.adoc

jessicax941 committed:
Remove unused import

jessicax941 committed:
Update UI class diagram

Dandford committed:
Update PPP

Dandford committed:
Resolving conflicts

jessicax941 committed:
Update Logic class diagram png file

yuhongtay committed:
final fix

Dandford committed:
Updating PPP

yuhongtay committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103-T16-4/main

# Conflicts:
#	src/main/java/seedu/guilttrip/model/GuiltTrip.java
#	src/main/java/seedu/guilttrip/storage/JsonAdaptedExpense.java
#	src/main/java/seedu/guilttrip/storage/JsonAdaptedIncome.java
#	src/main/java/seedu/guilttrip/storage/JsonAdaptedWish.java
#	src/test/java/seedu/guilttrip/logic/commands/FindExpenseCommandTest.java

yuhongtay committed:
Update UserGuide.adoc

jessicax941 committed:
Remove redundant files, update Model class diagram

Dandford committed:
Updating Pictures

yeejian-tan committed:
Update some links, minor User Stories editing

yuhongtay committed:
Update DeveloperGuide.adoc

yuhongtay committed:
fix TimeUtil

yuhongtay committed:
Merge branch 'master' of https://github.com/yuhongtay/main

Dandford committed:
Merge pull request #296 from SwampertX/userguide

UserGuidie: AutoSuggest and AutoExpense

yuhongtay committed:
Update DeveloperGuide.adoc

remove trailing  whitespace

jessicax941 committed:
Fix merge conflicts

jessicax941 committed:
Fix merge conflicts

rongrongrr committed:
Merge pull request #17 from AY1920S1-CS2103-T16-4/master

Merge Updates

rongrongrr committed:
Edit HistoryCommand

Dandford committed:
Merge pull request #294 from jessicax941/update-DG

Update UI class diagram, UI section in DG, add PPP

Dandford committed:
Final changes to PPP

rongrongrr committed:
Merge branch 'master' of https://github.com/rongrongrr/main

# Conflicts:
#	src/main/java/seedu/guilttrip/logic/commands/editcommands/EditBudgetCommand.java
#	src/main/java/seedu/guilttrip/logic/parser/editcommandparsers/EditBudgetCommandParser.java
#	src/main/java/seedu/guilttrip/ui/MainWindow.java

yuhongtay committed:
Update UserGuide.adoc

yuhongtay committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103-T16-4/main

# Conflicts:
#	src/main/java/seedu/guilttrip/ui/MainWindow.java

yuhongtay committed:
Merge branch 'master' of https://github.com/yuhongtay/main

yuhongtay committed:
Update UserGuide.adoc

jessicax941 committed:
Fix merge conflicts

Dandford committed:
Checking User Guide

yuhongtay committed:
Update UserGuide.adoc

Dandford committed:
Fix changes in User Guide

Dandford committed:
Merge pull request #292 from yuhongtay/master

Reminders

Dandford committed:
Merge branch 'master' of github.com:AY1920S1-CS2103-T16-4/main

yeejian-tan committed:
Avoid commiting empty guilt trip

Dandford committed:
Fixed my mistake

yeejian-tan committed:
Merge branch 'master' into last-fix

yeejian-tan committed:
Add DeveloperGuide for AutoExpense and AutoSuggest (#181)

* Add DeveloperGuide for AutoExpense

* Write DG for AutoSuggest

* Create sequence and activity diagrams for autoexp

* Done with DevGuide diagrams (Auto)

* Be more welcoming!

Save welcome message for startup.

yeejian-tan committed:
Merge branch 'master' into last-fix

rongrongrr committed:
Stop AutoExpenseUpdater from committing to GuiltTrip

rongrongrr committed:
Edit HistoryCommand

jessicax941 committed:
Merge branch 'master' into master

Dandford committed:
Merge branch 'master' of github.com:AY1920S1-CS2103-T16-4/main

Dandford committed:
Update branch

Dandford committed:
Merge pull request #297 from SwampertX/last-fix

Avoid commiting empty guilt trip

Dandford committed:
Merge branch 'master' of github.com:AY1920S1-CS2103-T16-4/main

Dandford committed:
Merge branch 'master' of github.com:Dandford/main

Dandford committed:
Merge pull request #298 from Dandford/master

Update The User Guide

jessicax941 committed:
Edit Income section of UG

jessicax941 committed:
Fix merge conflicts

jessicax941 committed:
Remove fxml warnings

jessicax941 committed:
Fix error on startup

jessicax941 committed:
Update version number to 1.4

jessicax941 committed:
Update AboutUs.adoc

Dandford committed:
Fix User Guide Errors

jessicax941 committed:
Update javadocs

jessicax941 committed:
Add list and listWish commands to UG

yeejian-tan committed:
SwampertX PPP (#299)

jessicax941 committed:
Merge branch 'master' into master

jessicax941 committed:
Fix merge conflicts

Dandford committed:
Fix User Guide Errors (#300)

jessicax941 committed:
Fix merge conflicts

Dandford committed:
Modifying PPP

Dandford committed:
Merge branch 'master' of github.com:Dandford/main

Dandford committed:
Push PPP

Dandford committed:
Merge pull request #295 from jessicax941/update-UG

Update UG on CRUD and UI section

Dandford committed:
Update PPP

Dandford committed:
Merge branch 'master' of github.com:AY1920S1-CS2103-T16-4/main

jessicax941 committed:
Merge pull request #302 from Dandford/master

Update PPP

rongrongrr committed:
Edit User Guide. Edit Developer Guide

rongrongrr committed:
Merge pull request #18 from AY1920S1-CS2103-T16-4/master

Merge Updates

rongrongrr committed:
Merge branch 'master' of https://github.com/rongrongrr/main

# Conflicts:
#	docs/UserGuide.adoc
#	src/main/java/seedu/guilttrip/commons/util/AutoExpenseUpdater.java

rongrongrr committed:
Fix Appveyor failure

Dandford committed:
Merge pull request #303 from rongrongrr/master

Edit HistoryCommand, UG, DG

jessicax941 committed:
Update PPP and User Guide

jessicax941 committed:
Merge remote-tracking branch 'upstream/master' into update-UG

jessicax941 committed:
Merge branch 'master' into update-UG

Dandford committed:
Merge pull request #307 from jessicax941/update-UG

Update PPP and User Guide

jessicax941 committed:
Fix error on startup

jessicax941 committed:
Merge remote-tracking branch 'upstream/master' into update-UG

Dandford committed:
FIxing User Guide (#306)

* Updating user guide small edits

* Still not showing chart. Push modified that

jessicax941 committed:
Fix remaining prefixes

jessicax941 committed:
Merge branch 'master' into update-UG

jessicax941 committed:
Update PPP

jessicax941 committed:
Merge remote-tracking branch 'upstream/master' into update-UG

jessicax941 committed:
Merge branch 'update-UG' of https://github.com/jessicax941/main into update-UG

Dandford committed:
Merge pull request #308 from jessicax941/update-UG

Update Ug and fix prefixes

rongrongrr committed:
Edit PPP (#309)

yuhongtay committed:
PPP (#305)

* Update UserGuide.adoc

* Create yuhong.adoc

* Update yuhong.adoc

* Update DeveloperGuide.adoc

* Update DeveloperGuide.adoc

* Update UserGuide.adoc

* Update yuhong.adoc

* Update yuhong.adoc

* Update UserGuide.adoc

* Update UserGuide.adoc

* Rename yuhong.adoc to yuhongtay.adoc

* Update yuhongtay.adoc

* Update UserGuide.adoc

* Update UserGuide.adoc

* Update yuhongtay.adoc

* Update AboutUs.adoc

* Update yuhongtay.adoc

* Update AboutUs.adoc

* Update yuhongtay.adoc

* Add files via upload

* Update UserGuide.adoc

yeejian-tan committed:
Some Docs Cosmetic Fixes (#304)

* Changes to website

1. Remove seedu header bar (refer to Mark)
2. Fix t/ to tg/ in UserGuide
3. Change John Doe to Tan Yee Jian

* Fix link and add Netlify Badge

yeejian-tan committed:
Link PPP to reposense (#315)
